idea_town/frontend/static-src/app/views/landing_page.js

```diff
@@ -0,0 +1,37 @@
+import app from 'ampersand-app';
+import AmpersandView from 'ampersand-view';
+import mustache from 'mustache';
+
+// TODO replace with an api endpoint that exposes idea town addon info
+const addonInfo = {
+  name: 'Idea Town',
+  url: '//localhost/codez/github/mozilla-idea-town-addon/@idea-town-addon-0.0.1.xpi'
```

Comment:
haha, yeah, I really didn't mean to leave the ol' `codez` directory in the final commit.

Until we have this provided by an API, I'll move the actual URL to settings.py, and have the server inject the path along with the [other unmentionables](https://github.com/mozilla/idea-town/blob/fc6070f/idea_town/frontend/templates/idea_town/frontend/index.html#L36-L43) we're currently injecting into the page itself.